src/store/plugins/click-on-map-management.plugin.js

```diff
@@ -8,8 +8,7 @@ import log from '@/utils/logging'
  *
  * @param {Vuex.Store} store
  * @param {ClickInfo} clickInfo Store mutation payload
- * @param {(WMSLayer | WMTSLayer | GeoJsonLayer | AggregateLayer)[]} visibleLayers All currently
- *   visible layers on the map
+ * @param {GeoAdminLayer[]} visibleLayers All currently visible layers on the map
```

Comment:
Should it not be AbstractLayer here ?

Reply:
Ok if I understand it correctly this function is only called for our layers and not for the externals as it then call our backend to get the info on the selected layer feature.